deploy-addresses/optimism-goerli.json

```diff
@@ -0,0 +1,4 @@
+{
+  "UniversalRouter": "0x4648a43B2C14Da09FdF82B161150d3F634f40491",
```

Comment:
these ended up being the same exact address despite different constructor params?

Reply:
yeah its deployed using CREATE not CREATE2

Reply:
ahh ok